deploy_notes_initial.md

```diff
@@ -16,13 +16,18 @@ These steps are intended to be run the first time a development or production co
       1. Using the SSH protocol: `git clone git@github.com:codeforgreenville/upstate_tech_cal_service.git`
       1. Or, using the HTTPS protcol: `git clone https://github.com/codeforgreenville/upstate_tech_cal_service.git`
    6. `cd upstate_tech_cal_service`
-   7. [Install Miniconda](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links) (Linux example) `curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh`
-   8. `sh miniconda.sh -b -p $HOME/miniconda`
-   9. `export PATH="$HOME/miniconda/bin:$PATH"`
-   10. `conda env create -f environment.yml`
-   1. Activate the application environment
-      1. `conda activate cal_service`
-      1. Or, the older way will work too: `source activate cal_service`
+   7. [Install Pipenv](https://pipenv.pypa.io/en/latest/#install-pipenv-today)  
+   For Linux: `curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python`  
```

Comment:
Probably best we just rely on the _Install Pipenv_ link above since there are numerous ways. For example, I'm on Ubuntu and I was able to run `pip install --user pipenv`, which is the first option listed in the install docs.

Reply:
Sounds great! I was a little confused if I should've had anything for Linux, only because there was in the Miniconda step. So I'm glad you commented on this!